integration/v7/isolated/app_command_test.go

```diff
@@ -253,6 +254,53 @@ applications:
 					Eventually(session).Should(Exit(0))
 				})
 			})
+
+			When("there is an active deployment", func() {
+				BeforeEach(func() {
+					helpers.WithHelloWorldApp(func(appDir string) {
+						Eventually(helpers.CF("push", appName, "-p", appDir, "-b", "staticfile_buildpack")).Should(Exit(0))
+					})
+				})
+
+				When("the deployment strategy is rolling", func() {
+					When("the deployment is in progress", func() {
+						It("displays the message", func() {
+							session := helpers.CF("restart", appName, "--strategy", "rolling")
+
+							session1 := helpers.CF("app", appName)
+							Eventually(session1).Should(Say("Rolling deployment currently DEPLOYING"))
+							Eventually(session).Should(Exit(0))
+							Eventually(session1).Should(Exit(0))
+						})
+					})
+					When("the deployment is cancelled", func() {
+						It("displays the message", func() {
+							helpers.CF("restart", appName, "--strategy", "rolling")
+							time.Sleep(3 * time.Second)
```

Comment:
**Suggestion** Is there a way to remove this sleep as it may cause flakes? 

I assume this is because the `cancel-deployment` fails until the deployment is active and deploying in cc which takes some time? Perhaps putting `cancel-deployment` in an Eventually block?

Reply:
Fixed